go/libkb/upak_loader.go

```diff
@@ -457,6 +458,42 @@ func (u *CachedUPAKLoader) LoadUserPlusKeys(ctx context.Context, uid keybase1.UI
 	return up, nil
 }
 
+// LoadKeyV2 looks through all incarnations for the user and returns the incarnation with the given
+// KID, as well as the Key data associated with that KID. It picks the first such incarnation if there
+// are multiple.
```

Comment:
Related to the comment about iteration order below. Maybe we should say "latest such"?